splitcolumn.py

```diff
@@ -80,19 +84,24 @@ def render(table, params):
     # Name the new columns
     conflict_colnames = list(table.columns)
     conflict_colnames.remove(colname)
-    uccolnames = gen_unique_clean_colnames(
+    uccolnames, warnings = gen_unique_clean_colnames_and_warn(
         ["%s %d" % (colname, i + 1) for i in range(len(newcols.columns))],
         existing_names=conflict_colnames,
     )
-    newcols.columns = [ucc.name for ucc in uccolnames]
-    # TODO warn about renames
+    newcols.columns = [ucc for ucc in uccolnames]
 
     if len(table.columns) > 1:
         # glue before, split, and after columns together
         # TODO test this
         colloc = table.columns.get_loc(colname)
         start = table.iloc[:, :colloc]
         end = table.iloc[:, colloc + 1 :]
-        return pd.concat([start, newcols, end], axis=1)
+        if warnings:
```

Comment:
These nested if-statements won't do. Please use a variable, so we only have one `if len(table.columns) > 1: ... else: ...` section (which sets a variable) and one `if warnings:` section (which returns).